app/src/main/java/org/schabi/newpipe/player/resolver/VideoPlaybackResolver.java

```diff
@@ -118,6 +122,11 @@ public MediaSource resolve(@NonNull final StreamInfo info) {
         }
     }
 
+    @Override
+    public boolean isVideoStreamVideoOnly() {
```

Comment:
I think that it should be noted that this is only set after ``resolve`` was called.

Maybe return an ``Optional`` instead? So that it's save that ``resolve`` was called.
Or maybe expand the returned data in ``resolve`` so that it contains this information?